specification/applicationinsights/resource-manager/readme.md

```diff
@@ -222,6 +222,31 @@ directive:
   
 ```
 
+### Tag: package-2022-01-11
+
+These settings apply only when `--tag=package-2022-01-11` is specified on the command line.
+
+``` yaml $(tag) == 'package-2022-01-11'
+input-file:
+  - Microsoft.Insights/stable/2015-05-01/aiOperations_API.json
+  - Microsoft.Insights/stable/2015-05-01/componentAnnotations_API.json
+  - Microsoft.Insights/stable/2015-05-01/componentApiKeys_API.json
+  - Microsoft.Insights/stable/2015-05-01/componentContinuousExport_API.json
+  - Microsoft.Insights/stable/2015-05-01/componentFeaturesAndPricing_API.json
+  - Microsoft.Insights/stable/2015-05-01/componentProactiveDetection_API.json
+  - Microsoft.Insights/stable/2015-05-01/componentWorkItemConfigs_API.json
+  - Microsoft.Insights/stable/2015-05-01/favorites_API.json
+  - Microsoft.Insights/stable/2015-05-01/webTestLocations_API.json
+  - Microsoft.Insights/stable/2015-05-01/webTests_API.json
+  - Microsoft.Insights/stable/2015-05-01/analyticsItems_API.json
+  - Microsoft.Insights/stable/2020-11-20/workbookTemplates_API.json
+  - Microsoft.Insights/stable/2021-03-08/myworkbooks_API.json
+  - Microsoft.Insights/stable/2021-08-01/workbooks_API.json
+  - Microsoft.Insights/preview/2018-05-01-preview/components_API.json
```

Comment:
>2018-05-01-preview/components_API.json [](http://example.com/codeflow?start=31&length=38)

should components_api.json be from a stable version? [https://github.com/Azure/azure-rest-api-specs/blob/main/specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/components_API.json](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/components_API.json)

once when it's been released it should be from the /stable

Reply:
There is actually an even newer stable version _stable/2020-02-02_, using that one.